WordPress/Classes/Extensions/Notifications/NotificationBlock+Interface.swift

```diff
@@ -81,12 +74,11 @@ extension NotificationBlock
         return attributedText(Constants.richHeaderTitleCacheKey)
     }
 
-    /**
-    *	@brief      Formats a NotificationBlock for usage into both, NoteBlockTextTableViewCell
-    *               and NoteBlockCommentTableViewCell.
-    *
-    *	@returns	An Attributed String for usage in both, comments and cell cells
-    */
+    /// Formats a NotificationBlock for usage into both, NoteBlockTextTableViewCell and
+    /// NoteBlockCommentTableViewCell.
+    ///
+    /// - Returns: An Attributed String for usage in both, comments and cell cells
```

Comment:
Is `cell cells` intentional?  Maybe use `regular cells` instead?  (not sure if that'd be correct)